app/src/utils/YupSchema.ts

```diff
@@ -117,6 +117,31 @@ yup.addMethod(
   }
 );
 
+yup.addMethod(
+  yup.string,
+  'isEndDateSameOrAfterStartDate',
+  function (
+    startDateName: string,
+    dateFormat: DATE_FORMAT = DATE_FORMAT.ShortDateFormat,
+    message = 'End date must be same or after start date'
```

Comment:
Subjective, but I think this might read a tiny bit better?
```suggestion
    message = 'End date must be on or after start date'
```